cleanlab_studio/studio/studio.py

```diff
@@ -142,6 +146,40 @@ def upload_from_url(
             schema_overrides=schema_overrides,
         )
 
+    def upload_from_bigquery(
+        self,
+        bigquery_project: str,
+        bigquery_dataset_id: str,
+        bigquery_table_id: str,
+        *,
+        schema_overrides: Optional[List[SchemaOverride]] = None,
+        **kwargs: Any,
+    ) -> str:
+        """
+        Uploads a dataset, from BigQuery, to Cleanlab Studio.
+
+        Args:
+            bigquery_project: BigQuery project ID.
+            bigquery_dataset_id: BigQuery dataset ID.
+            bigquery_table_id: BigQuery table ID.
+            schema_overrides: Optional list of overrides you would like to make to the schema of your dataset. If not provided, all columns will be untyped. Format defined [here](/guide/concepts/datasets/#schema-updates).
+        """
+        if isinstance(schema_overrides, dict):
+            # TODO: link to documentation for schema override format
```

Comment:
nit: just noticed this also exists in the other places we have schema_overrides, but I think this TODO is no longer relevant since we have the link in the docstring